packages/terra-form-select/CHANGELOG.md

```diff
@@ -5,10 +5,15 @@ Unreleased
 ----------
 ### Added
 * Create dropdown that announces no search results to screen reader
+* New CSS custom property: `--terra-form-select-disabled-placeholder-color`
```

Comment:
Should we consider this a breaking change?

Reply:
I was thinking along the same lines here. It's adding a theme variable for an existing colored area instead of adding a new color.